discovery-provider/alembic/trigger_sql/handle_supporter_rank_ups.sql

```diff
@@ -30,9 +31,10 @@ begin
       )
     on conflict do nothing;
 
-    if new.rank = 1 then 
-      select sender_user_id into dethroned_user_id from supporter_rank_ups where rank=1 order by slot desc limit 1;
-      if dethroned_user_id is not NULL then
+    if new.rank = 1 then
+      select sender_user_id into dethroned_user_id from supporter_rank_ups where rank=1 and receiver_user_id=new.receiver_user_id order by slot desc limit 1;
+      select true into has_previous_supporter from supporter_rank_ups where rank=new.receiver_user_id and receiver_user_id=1 HAVING COUNT(*) >= 2;
```

Comment:
not sure if i am missing context but in what case would `rank=new.receiver_user_id` ever happen? is it supposed to be `rank=1 and receiver_user_id=new.receiver_user_id`?

Reply:
yea yea def a bug, must have been bc I was testing with user id 1- thx great catch

Reply:
You are right the variables are switched - will fix